libs/ledger-live-common/src/families/cosmos/js-signOperation.ts

```diff
@@ -2,15 +2,15 @@ import type { Transaction } from "./types";
 import { Observable } from "rxjs";
 import { withDevice } from "../../hw/deviceAccess";
 import { encodeOperationId } from "../../operation";
-import { LedgerSigner } from "@cosmjs/ledger-amino";
-import { stringToPath } from "@cosmjs/crypto";
-import { buildTransaction, postBuildTransaction } from "./js-buildTransaction";
+import { txToMessages, buildTransaction } from "./js-buildTransaction";
 import BigNumber from "bignumber.js";
 import { makeSignDoc } from "@cosmjs/launchpad";
```

Comment:
I think the "@cosmjs/launchpad" is legacy.
you can do
```
import { serializeSignDoc, makeSignDoc } from "@cosmjs/amino";
```
and you can replace the "import { sortObjectKeysDeeply } from "./helpers";" with "serializeSignDoc" 

and don't have to maintain sortObjectKeysDeeply by ourselves

Reply:
Nice catch

https://github.com/cosmos/cosmjs/blob/main/packages/amino/src/signdoc.ts#L35

Reply:
Done